### PR TITLE
Jroach/fix demo

### PR DESCRIPTION
**Pull request by jeroach**
Minor tweaks to fix the demo and add a screenshot to the readme

### CODE REVIEW

**Comment by jeroach** [comment]
Just fixed a few links now that the component is correctly published: https://www.webcomponents.org/element/BrightspaceUI/navigation

File: README.md [hunk 1]
```diff
@@ -62,15 +62,17 @@ Wrapper components that packages up all other navigation components for ease of
 </head>
 ```
 
-Solid colour band that runs along the top of the navigational header
+Solid colour band that runs along the top of the navigational header.
+
+![screenshot of navigation band](/screenshots/navigation-band.png?raw=true)
```
Comment (svanherk) [inline README.md]: I _think_ GitHub is case-sensitive and won't find this due to the different extension casing.

**Comment by jeroach** [comment]
wtf edge... https://travis-ci.org/BrightspaceUI/navigation

**Comment by jeroach** [comment]
Deleted and re-added since renaming locally didn't change the extension case in GitHub

File: README.md (hunk 1 above)
Comment (jeroach) [reply README.md]: Thanks, fixed!

**Pull request merged**